v2.8.0 Breaks with ansi-to-html-umd

2.7.1 works great.
After npm installing with 2.8.0:
```
(webpack)-hot-middleware/client.js?path=http://localhost:3001/__webpack_hmr
[0] Module not found: Error: Cannot resolve module 'ansi-to-html-umd' in /project/node_modules/webpack-hot-middleware
[0] resolve module ansi-to-html-umd in /project/node_modules/webpack-hot-middleware
[0]   looking for modules in /project/src
[0]     /project/src/ansi-to-html-umd doesn't exist (module as directory)
[0]     resolve 'file' ansi-to-html-umd in /project/src
[0]       resolve file
[0]         /project/src/ansi-to-html-umd doesn't exist
[0]         /project/src/ansi-to-html-umd.json doesn't exist
[0]         /project/src/ansi-to-html-umd.js doesn't exist
[0]         /project/src/ansi-to-html-umd.jsx doesn't exist
[0]   looking for modules in /project/node_modules
[0]     /project/node_modules/ansi-to-html-umd doesn't exist (module as directory)
[0]     resolve 'file' ansi-to-html-umd in /project/node_modules
[0]       resolve file
[0]         /project/node_modules/ansi-to-html-umd doesn't exist
[0]         /project/node_modules/ansi-to-html-umd.json doesn't exist
[0]         /project/node_modules/ansi-to-html-umd.js doesn't exist
[0]         /project/node_modules/ansi-to-html-umd.jsx doesn't exist
[0]   looking for modules in /project/node_modules/webpack-hot-middleware/node_modules
[0]     /project/node_modules/webpack-hot-middleware/node_modules/ansi-to-html-umd doesn't exist (module as directory)
[0]     resolve 'file' ansi-to-html-umd in /project/node_modules/webpack-hot-middleware/node_modules
[0]       resolve file
[0]         /project/node_modules/webpack-hot-middleware/node_modules/ansi-to-html-umd doesn't exist
[0]         /project/node_modules/webpack-hot-middleware/node_modules/ansi-to-html-umd.json doesn't exist
[0]         /project/node_modules/webpack-hot-middleware/node_modules/ansi-to-html-umd.js doesn't exist
[0]         /project/node_modules/webpack-hot-middleware/node_modules/ansi-to-html-umd.jsx doesn't exist
[0] [/project/src/ansi-to-html-umd]
[0] [/project/src/ansi-to-html-umd]
[0] [/project/src/ansi-to-html-umd.json]
[0] [/project/src/ansi-to-html-umd.js]
[0] [/project/src/ansi-to-html-umd.jsx]
[0] [/project/node_modules/ansi-to-html-umd]
[0] [/project/node_modules/ansi-to-html-umd]
[0] [/project/node_modules/ansi-to-html-umd.json]
[0] [/project/node_modules/ansi-to-html-umd.js]
[0] [/project/node_modules/ansi-to-html-umd.jsx]
[0] [/project/node_modules/webpack-hot-middleware/node_modules/ansi-to-html-umd]
[0] [/project/node_modules/webpack-hot-middleware/node_modules/ansi-to-html-umd]
[0] [/project/node_modules/webpack-hot-middleware/node_modules/ansi-to-html-umd.json]
[0] [/project/node_modules/webpack-hot-middleware/node_modules/ansi-to-html-umd.js]
[0] [/project/node_modules/webpack-hot-middleware/node_modules/ansi-to-html-umd.jsx]
[0]  @ (webpack)-hot-middleware/client.js?path=http://localhost:3001/__webpack_hmr 82:11-38
[0] ./src/helpers/ApiClient.js
```

--- a/package.json
+++ b/package.json
@@ -15,10 +15,10 @@
   "license": "MIT",
   "dependencies": {
     "querystring": "^0.2.0",
-    "strip-ansi": "^3.0.0"
+    "strip-ansi": "^3.0.0",
+    "ansi-to-html-umd": "^0.4.2"
   },
   "devDependencies": {
-    "ansi-to-html-umd": "^0.4.2",
     "express": "^4.13.3",
     "mocha": "^2.3.2",
     "supertest": "^1.1.0",

